examples/oauth/src/main.rs

```diff
@@ -256,3 +304,32 @@ where
         Ok(user)
     }
 }
+
+// Use anyhow, define error and enable '?'
+// For a simplified example of using anyhow in axum check /examples/anyhow-error-response
+#[derive(Debug)]
+struct AppError(anyhow::Error);
+
+// Tell axum how to convert `AppError` into a response.
+impl IntoResponse for AppError {
+    fn into_response(self) -> Response {
+        tracing::error!("Application error: {:#}", self.0);
+
+        (
+            StatusCode::INTERNAL_SERVER_ERROR,
+            "Something went wrong".to_string(),
```

Comment:
nit: You don't need to call `.to_string()` here since `IntoResponse` is implemented for `&'static str`.

```suggestion
            "Something went wrong",
```